include/avif/avif.h

```diff
@@ -1128,8 +1128,10 @@ AVIF_API avifResult avifEncoderAddImageGrid(avifEncoder * encoder,
                                             avifAddImageFlags addImageFlags);
 AVIF_API avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output);
 
-// Codec-specific, optional "advanced" tuning settings, in the form of string key/value pairs.
-// key must be non-NULL, but passing a NULL value will delete that key, if it exists.
+// Codec-specific, optional "advanced" tuning settings, in the form of string key/value pairs,
+// to be sent to codec at the next avifEncoderAddImage() call.
+// See the codec documentation to know if a setting is permanent or applied only to the next frame.
```

Comment:
Do you know of any codec-specific option that applies only to the next frame? This kind of option seems inconvenient to use.

I think the `flags` parameter of `aom_codec_encode()` is intended for options that apply only to the next frame.